File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.cpp
#include<bits/stdc++.h>
using namespace std;
vector<int>v;
string solution(string s) {
    string answer = "";
    string temp="";
    for (auto cur :s)
    {
        if (cur==' ')
        {
            v.push_back(stoi(temp));
            temp="";
            continue;
        }
        temp+=cur;
    }
    v.push_back(stoi(temp));
    sort(v.begin(),v.end());
    int l=v.size();
    answer=to_string(v[0])+' '+to_string(v[l-1]);
    return answer;
}

File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.java
class Solution {
    public String solution(String s) {
        String answer = "";
        String t[] = s.split(" ");
        int mx=-987654321;
        int mi=987654321;
        for (String cur : t)
        {
            int a=Integer.parseInt(cur);
            mx=Math.max(mx,a);
            mi=Math.min(mi,a);
        }
        return mi+" "+mx;
    }
}